src/bin/text/index.mld

```diff
@@ -5,9 +5,11 @@
 The alt-ergo package installs the {e alt-ergo} binary, whose documentation
 is available through the {e --help} option.
 
+See also the {{:https://ocamlpro.github.io/alt-ergo/}language documentation}.
+
```

Comment:
Why do you add a link here? There is very same link on the sidebar.

Reply:
This is the odoc for the alt-ergo package (this page: https://ocamlpro.github.io/alt-ergo/odoc/dev/alt-ergo/index.html ), I don't see a link to the language doc in the sidebar?

(Sorry, this probably should have been in a separate PR, I just did it as I was in the doc folder)

Reply:
Oh sorry, I thought it was in the sphinx documentation. You don't need to do a separate PR for such a small modification.